cloudbuild.yaml

```diff
@@ -21,6 +21,8 @@ steps:
   args:
   - -exc
   - |
+    bazel build ...
```

Comment:
Instead of doing this, we can also just add a new step that uses bazel cloud build directly. See https://github.com/GoogleCloudPlatform/cloud-builders/tree/master/bazel.

Another benefit is that this can run in parallel.

Reply:
Thanks for the link. Added a new step.